src/test/java/org/apache/drillui/test/framework/initial/PropertiesConst.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.drillui.test.framework.initial;
+
+public class PropertiesConst {
+    public static String ADMIN_1_NAME=TestProperties.get("ADMIN_1_NAME");
```

Comment:
```suggestion
    public static final String ADMIN_1_NAME=TestProperties.get("ADMIN_1_NAME");
```

Also could you please add constants for other properties as well?

Reply:
@agozhiy  Easy. Changes added.